es-ds-docs/pages/molecules/radio-button.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+    <div>
+        <h1>
+            Radio button
+        </h1>
+        <p class="mb-500">
+            Uses <a
+                href="https://v3.primevue.org/radiobutton/"
+                target="_blank">
+                PrimeVue RadioButton
+            </a>
+        </p>
+
+        <div class="my-500">
+            <h2>
+                Inline
+            </h2>
+            <p>
+                Please choose your favorite fruit.
+            </p>
+            <div>
+                <div v-for="fruit in fruits" class="custom-control custom-control-inline custom-radio">
+                    <RadioButton
+                        class="custom-control-input"
+                        inputClass="custom-radio-input"
+                        v-model="selectedFruit"
+                        :inputId="`${fruit.key}-inline`"
+                        name="inline"
+                        :value="fruit.name" />
```

Comment:
The `value` for each radio button should correspond to the "key" of each fruit (so this should be `:value="fruit.key"`). Right now, setting `selectedFruit` does not do anything because of the mismatch.